VectorGatherMaskFoldingTest fails for s390x and riscv jdk 23

Logs:
```
[2024-09-16T17:39:11.476Z] Caused by: java.lang.RuntimeException: assertNotEquals: expected [1] to not equal [1]
[2024-09-16T17:39:11.476Z] 	at jdk.test.lib.Asserts.fail(Asserts.java:691)
[2024-09-16T17:39:11.476Z] 	at jdk.test.lib.Asserts.assertNotEquals(Asserts.java:451)
[2024-09-16T17:39:11.476Z] 	at jdk.test.lib.Asserts.assertNotEquals(Asserts.java:435)
[2024-09-16T17:39:11.476Z] 	at compiler.vectorapi.VectorGatherMaskFoldingTest.testTwoLongVectorLoadGatherMaskedNotEqualMask(VectorGatherMaskFoldingTest.java:227)
[2024-09-16T17:39:11.476Z] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[2024-09-16T17:39:11.476Z] 	... 7 more
```

From:
https://ci.adoptium.net/job/Grinder/10882/consoleText

--- a/openjdk/excludes/ProblemList_openjdk21.txt
+++ b/openjdk/excludes/ProblemList_openjdk21.txt
@@ -379,7 +379,6 @@ compiler/codecache/MHIntrinsicAllocFailureTest.java https://bugs.openjdk.org/bro
 compiler/whitebox/ForceNMethodSweepTest.java https://bugs.openjdk.org/browse/JDK-8265181 linux-arm
 compiler/codegen/aes/Test8292158.java https://github.com/adoptium/aqa-tests/issues/4470 linux-s390x
 compiler/codegen/aes/TestAESMain.java https://github.com/adoptium/aqa-tests/issues/4470 linux-s390x
-compiler/vectorapi/VectorGatherMaskFoldingTest.java https://bugs.openjdk.org/browse/JDK-8333248 linux-s390x,linux-riscv64
 
 ############################################################################
 

--- a/openjdk/excludes/ProblemList_openjdk23.txt
+++ b/openjdk/excludes/ProblemList_openjdk23.txt
@@ -389,8 +389,6 @@ compiler/codecache/MHIntrinsicAllocFailureTest.java https://bugs.openjdk.org/bro
 compiler/whitebox/ForceNMethodSweepTest.java https://bugs.openjdk.org/browse/JDK-8265181 linux-arm
 compiler/codegen/aes/Test8292158.java https://github.com/adoptium/aqa-tests/issues/4470 linux-s390x
 compiler/codegen/aes/TestAESMain.java https://github.com/adoptium/aqa-tests/issues/4470 linux-s390x
-compiler/vectorapi/VectorGatherMaskFoldingTest.java https://bugs.openjdk.org/browse/JDK-8333248 linux-s390x,linux-riscv64
-
 ############################################################################
 
 # jdk_jfr

